.github/workflows/publish.yml

```diff
@@ -14,6 +14,18 @@ jobs:
       with:
         node-version: 14.x
         cache: 'yarn'
+    - uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: "us-gov-west-1"
+
+    - name: Get bot token from Parameter Store
+      uses: marvinpinto/action-inject-ssm-secrets@v1.2.1
+      with:
+        ssm_parameter: /devops/VA_VSP_BOT_NPM_TOKEN
+        env_variable_name: VA_VSP_BOT_NPM_TOKEN
```

Comment:
This pattern is copied from elsewhere in [`content-build`](https://github.com/department-of-veterans-affairs/content-build/blob/5f6b464ece2b595b3bd9af29c1b773bda95700e4/.github/workflows/continuous-integration.yml#L252-L265)